MSAL/src/native_auth/public/state_machine/MSALNativeAuthRequiredAttributes.swift

```diff
@@ -24,11 +24,16 @@
 
 import Foundation
 
+/// Class that defines the structure of a Required Attribute
 @objc
-public class MSALNativeAuthRequiredAttributes: NSObject {
+public class MSALNativeAuthRequiredAttribute: NSObject {
+    /// The name of the Required Attribute
     public let name: String
+    /// The type of the Required Attribute
     public let type: String
+    /// Specifies if the Attribute is required or not
     public let required: Bool
```

Comment:
Now that I'm reading the name of this class and its documentation... does this variable make sense in this class? It's supposed to always be `true`?

Reply:
Asked APi team

Reply:
Let's follow up with them, but I'm resolving this to avoid blocking the PR